src/data/cryptocurrencies.js

```diff
@@ -1519,6 +1519,28 @@ const cryptocurrenciesById = {
     ],
     txExplorers: ["https://tzscan.io/$hash"]
   },
+  tomo: {
+    id: "tomo",
+    coinType: 889,
+    name: "TomoChain",
+    managerAppName: "TomoChain",
+    ticker: "TOMO",
+    scheme: "tomo",
+    color: "#FF9933",
+    family: "ethereum",
+    ethereumLikeInfo: {
+      chainId: 88
+    },
+    blockAvgTime: 2,
```

Comment:
the blockchains update every 2 seconds !?

Reply:
yep, create new block every 2 seconds (150 block producer - masternodes, see https://stats.tomochain.com)

Reply:
neat